src/openrct2-ui/windows/Multiplayer.cpp

```diff
@@ -487,7 +492,7 @@ static void WindowMultiplayerPlayersResize(rct_window* w)
 {
     window_set_resize(w, 420, 124, 500, 450);
 
-    w->no_list_items = network_get_num_players();
+    w->no_list_items = (IsServerPlayerInvisible() ? network_get_num_visible_players() : network_get_num_players());
```

Comment:
`network_get_num_visible_players()` already accounts for this.

```suggestion
    w->no_list_items = network_get_num_visible_players();
```